src/gh/components/DF_cloud_mesh_distance/code.py

```diff
@@ -43,7 +41,11 @@ def RunScript(self,
             rh_mesh_target_list = [joint.to_mesh(i_analysis_resolution) for joint in i_assembly._all_joints]
         elif len(i_assembly.all_joint_faces) == len(i_cloud_list):
             ghenv.Component.Message = "Per Joint Face"  # noqa: F821
-            rh_mesh_target_list = [joint_face.to_mesh() for joint_face in i_assembly.all_joint_faces]
+            rh_mesh_target_list = []
+            for joint in i_assembly.all_joints:
+                for face in joint.faces:
+                    face = face.to_mesh()
```

Comment:
Small comment here and perhaps not directly related to the PR.. for the case of beam or joint we are using the `joint.to_mesh()` and `beam.to_mesh()` that get the input of the `MaximumEdgeLength` which the user directly changes when meshing the geometry. This is not the case now for the face of each joint where `Rhino.Geometry.MeshingParameters.QualityRenderMesh` is always used.